src/components/Search/SearchView.tsx

```diff
@@ -22,8 +23,14 @@ function SearchView(props: SearchViewProps) {
     useFocusInput();
 
   const navigate = useNavigate();
-  const goToCompareResultsPage = () => {
-    navigate('/compare-results', { replace: false });
+
+  const goToCompareResultsPage = (selectedRevisions: Revision[]) => {
+    const revisions = [...selectedRevisions.map(rev => truncateHash(rev.revision))];
+    const repos = [...selectedRevisions.map(rev => rev.repository_id)];
+    navigate({
+      pathname: '/compare-results', 
+      search: `?revs=${revisions.toString()}&repos=${repos.toString()}`,
```

Comment:
optional: it may be clearer to the reader if you use the `join` operation: `revisions.join(',')`

Reply:
I don't have a preference either way. although I didn't know that `array.toString` would automatically comma delimit it--I learned something :)

Reply:
done